misc/python/requirements-dev.txt

```diff
@@ -2,3 +2,6 @@ black==19.10b0
 mypy==0.770
 pdoc3==0.8.1
 pytest==5.4.2
+
+# these requirements are used only in internal scripts, not for general use of materialize tools
+requests>=2.24.0,<2.25.0
```

Comment:
I'm not sure I understand how `requests` is different from the other stuff in either of these files?

Reply:
it's different because it is necessary only for the bintray code -- but not necessary for non-materialize folks interacting with just our demos. That is the audience that I think of as being the reason that we try to keep the requirements slim.

Reply:
Oh I just realized that there bintray script must be used by ci, not humans, and that it is therefore probably not different.